frontend/src/styles/signin.css

```diff
@@ -0,0 +1,16 @@
+/* Place jumbotron in the center of the screen, with slight transparency. */
+.jumbotron-signin {
+  background-color: rgba(255, 255, 255, 0.8);
```

Comment:
can you leave a /*comment*/ telling us what color this is?

Reply:
I've updated the jumbotron comments. 👍